supplementary_style_guide/style_guidelines/structure.adoc

```diff
@@ -1,6 +1,66 @@
 [[structure]]
 = Structure
 
+[[abstracts-or-shortdesc]]
+== Abstracts or short descriptions
+
+To help readers find the information that they need or to confirm that they are in the right place, every module and assembly must include a short description. In CCS, we have referred to this content as an abstract or introductory paragraph. We plan to standardize on the term *short description*.
+
+This description is usually at least 2-3 sentences long, and you can scan it in a few seconds. It exists between the title and the main content, connecting them and providing context and disambiguation.
+
+Follow these guidelines when writing short descriptions:
+
+[discrete]
+=== Content
+
+* Include user intent:
+** *What* users must do. This content is similar to what is in the title but should not just repeat the same information.
+** *Why* users must complete an action. You must build from the information in the title. This content helps users understand why completing an action is important or beneficial to them.
+*** If the title of your module is “Creating a Kafka instance in OpenShift Streams for Apache Kafka”, your short description could begin something like this: “To connect OpenShift Streams for Apache Kafka to other applications or services [*why*], create and configure a Kafka instance in the web console [*what*].”
```

Comment:
Recommend removing the example, as the what and why portions are self evident and the example is reused below.

Reply:
Yes, maybe I could remove this here and add **[what]** and **[why]** in the full version of the short description.